app/assets/javascripts/discourse/app/components/post-list/index.gjs

```diff
@@ -0,0 +1,85 @@
+/**
+ * A component that renders a list of posts
+ *
+ * @component PostList
+ *
+ * @args {Array<Object>} posts - The array of post objects to display
+ * @args {Function} fetchMorePosts - A function that fetches more posts. Must return a Promise that resolves to an array of new posts.
+ * @args {String} emptyText (optional) - Custom text to display when there are no posts
+ * @args {String|Array} additionalItemClasses (optional) - Additional classes to add to each post list item
```

Comment:
Can't we use ...attributes instead?

Reply:
I don't think so - those attributes would apply to the `.post-list` wrapper, not the individual `<PostListItem`.